index.js

```diff
@@ -88,6 +88,8 @@ module.exports = function () {
     if (!system.global.hasOwnProperty('monitor')) { system.global.monitor = true }
     if (!system.global.hasOwnProperty('monitor_excludes')) { system.global.monitor_excludes = [] }
     if (!system.global.hasOwnProperty('dns_enabled')) { system.global.dns_enabled = false }
```

Comment:
Just because some of those properties are boolean so a falseiness test wouldn't work... Can you merge and then send me a PR with cleaned up code ?